NAMESPACE

```diff
@@ -6,4 +6,3 @@ export(easy_remove_x_axis)
 export(easy_remove_y_axis)
 export(easy_rotate_x_labels)
 import(ggplot2)
```

Comment:
Did you re-build before this push? Shouldn't `import(rlang)` show up still?

Reply:
I get muddled with this all the time. We have no specific imports of `rlang` so it's not imported wholesale, is that right? The namespace isn't available by default, so any functions need to be prefixed with `rlang::`. This is fine.

Reply:
Yeah I switched over to doing rlang:: because I noticed you did that in code somewhere and thought we should be consistent.